src/webinit.m

```diff
@@ -16,14 +16,22 @@ D INSTALLRO("https://github.com/shabiel/M-Web-Server/releases/download/1.1.0/mws
  ;
  ; Load the URLs
  D LOADHAND
- ; 
+ ;
  ; Set the home directory for the server
  N % S %=$$HOMEDIR
  I '% QUIT
  ;
  ; Set start port
  N PORT S PORT=$$PORT()
  I PORT=0 QUIT
+ S ^%webhttp(0,"port")=PORT
+ ;
+ ; Set CORS config
```

Comment:
Sam will have to review this routine as I don't use it.

Reply:
Sure